test_pull_requests

```diff
@@ -1183,6 +1354,7 @@ popd
         # we need to take the correct external actions if necessary and update the bot
         # comment to reflect the new state
         if submit_test_job
+          delete_comment_with_prefix(id, base_repo, flake_denied_prefix(base_repo, settings['name']), comments)
```

Comment:
I thought at this point we've determined that we are submitting a test job, so the first time we decide we are going to (re)submit the test job we know we can remove this comment.